plugins/kontent-plugin/scripts/setup.js

```diff
@@ -0,0 +1,109 @@
+const chalk = require("chalk")
+const argv = require("yargs-parser")(process.argv.slice(2))
+const fs = require("fs")
+const path = require("path")
+const inquirer = require("inquirer")
+const { ImportService, ZipService } = require("@kentico/kontent-backup-manager")
+const {
+  FileService,
+} = require("@kentico/kontent-backup-manager/dist/cjs/lib/node")
+
+const importData = async (projectId, managementApiKey) => {
+  const fileService = new FileService({
+    enableLog: true,
+  })
+
+  const zipFile = await fileService.loadFileAsync("data")
+
+  const zipService = new ZipService({
+    context: "node.js",
+    enableLog: true,
+  })
+
+  const importService = new ImportService({
+    onImport: (item) => {
+      console.log(`Imported: ${item.title} | ${item.type}`)
+    },
+    canImport: {
+      asset: (item) => true,
+      contentType: (item) => true,
+      assetFolder: (item) => true,
+      contentItem: (item) => true,
+      contentTypeSnippet: (item) => true,
+      language: (item) => true,
+      languageVariant: (item) => true,
+      taxonomy: (item) => true,
+    },
+    enablePublish: true,
+    projectId: projectId,
+    apiKey: managementApiKey,
+    enableLog: true,
+    fixLanguages: true,
+  })
+
+  const importData = await zipService.extractZipAsync(zipFile)
+
+  await importService.importFromSourceAsync(importData)
+}
+
+console.log(`
+  To use this starter, please create a new project in app.kentico.ai
+  and provide us with credentials need to set it up.
+  The required keys can be found in Project settings -> API KEYS and
+  you will need:
+
+    1. ${chalk.blue("Project ID")}
+    2. ${chalk.blue(
+      "API Management key"
+    )} needed for importing the content into the project
+
+  Everything ready? Let's do it!
+`)
+
+const questions = [
+  {
+    name: "projectId",
+    message: "Project ID",
+    when: !argv.projectId && !process.env.KONTENT_PROJECT_ID,
+  },
+  {
+    name: "managementApiKey",
+    when: !argv.managementApiKey,
+    message: "Your Content Management API access token",
+  },
+]
+
+inquirer
+  .prompt(questions)
+  .then(({ projectId, managementApiKey }) => {
+    // write env vars to .env.development & .env.production
+    const dotenv = [
+      `# All environment variables will be sourced`,
+      `# and made available to gatsby-config.js, gatsby-node.js, etc.`,
+      `# Do NOT commit this file to source control`,
+      `KONTENT_PROJECT_ID="${projectId}"`,
+    ].join("\n")
+
+    const configFiles = [".env.development", ".env.production"]
+      .map((name) => path.join(__dirname, "..", name))
+      .forEach((filename) => {
+        fs.writeFileSync(filename, dotenv, "utf8")
+      })
+
+    console.log(`.env files written`)
+
+    return { projectId, managementApiKey }
+  })
+  .then(({ projectId, managementApiKey }) => {
+    importData(projectId, managementApiKey)
```

Comment:
Upon a subsequent run of this `setup` script with adding the `return`, I received the following error:

![Uploading Screen Shot 2022-06-29 at 12.51.10 PM.png…]()

Reply:
After hacking the source of `file.service.js` to resolve the correct path to the `data.zip`, I received a 401 from the kontent management API:

<img width="1038" alt="Screen Shot 2022-06-29 at 12 58 16 PM" src="https://user-images.githubusercontent.com/1790506/176420925-d53cdb45-75a1-4850-adb0-ff58a6c6a89a.png">

Reply:
Thanks for finding that! I have adjusted the script so it can be executed from any directory. However, I did not get the  401 error as you. What comes to my mind is, can it be that you don't have an empty project?

Reply:
After testing out the setup script again, I am not received the same issue as shown before. However I am received a different error unfortunately.

I have ran this twice (I cleared out the Kontent.ai content model between runs) and received the same error both times:

<img width="1343" alt="Screen Shot 2022-09-05 at 5 44 47 PM" src="https://user-images.githubusercontent.com/1790506/188524613-45742301-c1a2-4180-bc35-5956034130d2.png">